app-web/src/components/Navbar/Navbar.js

```diff
@@ -91,7 +91,7 @@ export const NavList = styled.ul`
 
 const AuthContainer = styled.div`
   padding: 10px;
-  background-color: #fafafa;
+  border-bottom: 1px solid #bbc7d5;
```

Comment:
why remove the bg?

Reply:
The logout button looks drastically different than the login button, after removing the background. 👍